src/python/gudhi/sklearn/rips_persistence.py

```diff
@@ -52,25 +52,25 @@ def __init__(
         homology_coeff_field: int = 11,
         n_jobs: Optional[int] = None,
     ):
-        """
-        Constructor for the RipsPersistence class.
-
-        Parameters:
-            homology_dimensions (int or list of int): The returned persistence diagrams dimension(s).
-                Short circuit the use of :class:`~gudhi.representations.preprocessing.DimensionSelector` when only one
-                dimension matters (in other words, when `homology_dimensions` is an int).
-            threshold (float): Rips maximal edge length value. Default is +Inf. Ignored if input_type is 'distance coo_matrix'.
-            input_type (str): Can be 'point cloud' when inputs are point clouds, 'full distance matrix',
-                'lower distance matrix' when inputs are lower triangular distance matrix (can be full square,
-                but the upper part will not be considered), or 'distance coo_matrix' for a distance matrix in SciPy's
-                sparse format, which should contain each edge at most once (avoid the symmetric) and no diagonal entry.
-                Default is 'point cloud'.
-            num_collapses (int|str): Specify the number of iterations of :func:`~gudhi.flag_filtration.edge_collapse.reduce_graph`
-                (edge collapse) to perform on the graph. Default value is 'auto'.
-            homology_coeff_field (int): The homology coefficient field. Must be a prime number. Default value is 11.
-            n_jobs (Optional[int]): Number of jobs to run in parallel. `None` (default value) means `n_jobs = 1` unless in a
-                joblib.parallel_backend context. `-1` means using all processors. cf.
-                https://joblib.readthedocs.io/en/latest/generated/joblib.Parallel.html for more details.
+        """Constructor for the RipsPersistence class.
+
+        :param homology_dimensions: The returned persistence diagrams dimension(s).
```

Comment:
I see you switched from Google-style to plain RST. Is that because it wasn't possible while sticking to Google-style?

Reply:
I was just following what is explained in [sphinx-autodoc-typehints documentation](https://github.com/tox-dev/sphinx-autodoc-typehints). Google style is also working, so I reverted to it on [52aa602](https://github.com/GUDHI/gudhi-devel/pull/1137/commits/52aa6022583a68c781ce925a904d7870c5206c62)